src/content/1.7/project/release/release-a-new-version-of-prestashop.md

```diff
@@ -0,0 +1,187 @@
+
+### 3.1 Merge the build branch on GitHub
+
+Open a Pull Request to merge the work you have carried out for the Build: updating the version number and modifying Changelog and Contributors files.
+
+This is the branch you created at step 3 "Create a new branch, commit your changed files" when building the ZIP archive.
+
+The Pull Request must target the `1.7.x.y` branch used for building the new release.
+
+When it is merged, **[tag the last commit][git-create-tag] on this branch**. Not the merge commit that was created by GitHub, but rather the one before. The one that has commit message "// Changelog [version]".
+
+### 3.2 Create GitHub release
+
+Open GitHub and go to ["Releases" page][core-github-releases], click "Draft a new release".
+
+- use the tag you created as a target
+- use the Changelog as description
+- attach the ZIP archive
+
+And publish the new release !
+
+You can update the Release Tracker GitHub issue: step "Release" is done.
+
+## 4. Make a new version on prestashop.com
+
+This information is private to PrestaShop company. It simply involves uploading the ZIP on the website for distribution.
+
+## 5. Configure Addons API for the new version
+
+This information is private to PrestaShop company.
+It aims to allow the Addons API to serve modules for the new version.
+
+## 6. Configure Internationalization API for the new version
+
+This information is private to PrestaShop company.
+It aims to allow the Internationalization API to serve localization packs for the new version.
+
+## 7. Update XML stream to allow auto-upgrade module to perform upgrades to the new version
+
+This information is private to PrestaShop company.
+
+It aims to update the XML document used by PrestaShop API for the upgrade process. Once updated, the [auto-upgrade module][autoupgrade-repository] is able to parse it and acknowledge the new available version.
+
+You can update the Release Tracker GitHub issue: step "Available for upgrade" is done.
+
+## 8. Create Docker images for the new version
+
+1. Checkout the [project][docker-repository] and create a new branch
+2. Modify the following files (see [this PR as example][docker-release-pr-example]:
+ - `versions.txt` - in which you add the new version
+ - `versions<major version>.txt` - the new release must be added at the end of the file
+ - `.travis.yml` - to test the image creation with the new release, you can update the test case corresponding to the same major version.
+3. Commit these changes
+4. Run `generate_all.sh`, all Dockerfile will (re)generated in the folder `images/`
+5. Commit these changes
+6. Create a PR to merge this branch into upstream
+7. PR must be approved and merged.
+
+Docker Hub is synchronized with the GitHub repository and will quickly provide the new images.
```

Comment:
Ping @PierreRambaud

Reply:
It's not synchronized yet, we still have to update images manually.

Reply:
Eh ? So when we did https://github.com/PrestaShop/docker/pull/210 https://github.com/PrestaShop/docker/pull/225 https://github.com/PrestaShop/docker/pull/218 you had to push the images?